index.html

```diff
@@ -85,8 +85,7 @@ <h2 class="element__name"></h2>
             <p class="popup__signature"></p>
         </div>
     </div>
-<script src="./js/validate.js"></script>
-<script src="./js/initial-сards.js"></script>
-<script src="./js/script.js"></script>
+<script type="module" src="./js/index.js"></script>
+<script type="module" src="./js/Card.js"></script>
```

Comment:
уверен что хочешь удалить? 
```suggestion
<script type="module" src="./js/Card.js"></script>
```

Reply:
да

Reply:
точно уверен